### ISSUE TITLE
ERR_INSECURE_RESPONSE

### ISSUE THREAD

**Issue by timsayshey** [prisma/graphql-playground#1064]
I'm trying to run the Electron app but I keep getting ERR_INSECURE_RESPONSE when it tries to connect to my local GraphQL server with a self signed certificate.
![image](https://user-images.githubusercontent.com/1148940/62340545-8fff7d80-b4a5-11e9-8b7f-39eb2a5bfa32.png)

This is probably because it's not running from 127.0.0.1. It's running from Minikube which runs on 192.168.99.100. Is there a way to whitelist this IP so I can use a self-signed certificate?

**Comment by sebastienbarre** [comment]
Indeed, would be great if it was configurable somewhere in the Settings...

**Comment by ppodila** [comment]
what would be the  command on windows 10. I see the same error while accessing https://localhost/graphql
